tests/utilities/mock/Utapi.js

```diff
@@ -0,0 +1,82 @@
+const express = require('express');
```

Comment:
This feels like a lot of code for a mock. Feels like we would need to add tests here as it has a fair bit of logic.